### PR TITLE
Typo fix in docs code sample

### PR DESCRIPTION
**Pull request by dmitryrn**

### CODE REVIEW

File: docs/usage/usage-guide.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ const todosReducer = createReducer([], {
         // Can still return an immutably-updated value if we want to
         return state.filter( (todo, i) => i !== action.payload.index)
     }
-}
+})
```
Comment (denisw) [inline docs/usage/usage-guide.md]: This is no typo. The `)` closes the argument list of the `createReducer()` call.

Comment (dmitryrn) [reply docs/usage/usage-guide.md]: So I just added it

Comment (denisw) [reply docs/usage/usage-guide.md]: Oops, sorry, I read the diff the wrong way around. 😅

**Comment by markerikson** [comment]
Thanks!

**Pull request merged**